test/features/deploy_kogito_runtime.feature

```diff
@@ -0,0 +1,125 @@
+# Disabled until we can integrate it as part of the pipeline, see KOGITO-2278
+@disabled
+Feature: Deploy Kogito Runtime
+
+  Background:
+    Given Namespace is created
+
+  @smoke
+  Scenario Outline: Deploy Simplest Scenario using Kogito Runtime
+    Given Kogito Operator is deployed
+    And Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven using profile "<profile>"
+    And Local example service "<example-service>" is deployed to image registry, image tag stored as variable "built-image"
+
+    When Deploy <runtime> example service using image in variable "built-image" with configuration:
+      | config | persistence | disabled |
+    
+    Then Kogito Runtime "<example-service>" has 1 pods running within 10 minutes
+    And Service "<example-service>" with process name "orders" is available within 2 minutes
+
+    @springboot
+    Examples:
+      | runtime    | example-service            | profile |
+      | springboot | process-springboot-example | default |
+
+    @quarkus
+    Examples:
+      | runtime    | example-service         | profile |
+      | quarkus    | process-quarkus-example | default |
+
+    @quarkus
+    @native
+    Examples:
+      | runtime    | example-service         | profile |
+      | quarkus    | process-quarkus-example | native  |
+
+#####
+
+  @persistence
+  Scenario Outline: Deploy Scenario with persistence using Kogito Runtime
+    Given Kogito Operator is deployed with Infinispan operator
+    And Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven using profile "<profile>"
+    And Local example service "<example-service>" is deployed to image registry, image tag stored as variable "built-image"
+
+    When Deploy <runtime> example service using image in variable "built-image" with configuration:
+      | config | persistence | enabled |
+    And Kogito Runtime "<example-service>" has 1 pods running within 10 minutes
+    And Start "orders" process on service "<example-service>" with body:
```

Comment:
are we sure here that the route will be fully available on first call (existing for sure) ? 
shouldn't we have a timeout to retry ?

Reply:
It never failed for me. But will add it just to be sure.